static/app/data/forms/projectGeneralSettings.tsx

```diff
@@ -72,6 +73,10 @@ export const fields: Record<string, Field> = {
         </PlatformWrapper>,
       ]),
     help: t('The primary platform for this project'),
+    // we need to look at the children to get the text
+    filterOption: createFilter({
+      stringify: option => `${option.label?.props?.children[1]} ${option.value}`,
```

Comment:
👎 to looking into `children`, let's look up the option in `platforms` with something like

```suggestion
      stringify: option => `${platforms.find(([id]) => id === option.value)[1]} ${option.value}`,
```

Reply:
@evanpurkhiser right, I can just look at the choices here, good call